extension/src/components/UpdateTreeNode.tsx

```diff
@@ -4,9 +4,10 @@ import { UpdateItem } from "./UpdateItem";
 
 interface UpdateTreeNodeProps {
 	node: UpdateTreeNode;
+	count?: number;
 }
 
-export function UpdateTreeNodeComponent({ node }: UpdateTreeNodeProps) {
+export function UpdateTreeNodeComponent({ node, count }: UpdateTreeNodeProps) {
```

Comment:
i really don't like this, so any advice would be appreciated

we want to tell the `UpdateItem` the count, but the `UpdateTreeNode` doesn't really care

if this is the right way of doing that, fair enough